align/primitive/main.py

```diff
@@ -421,28 +421,22 @@ def generate_primitive(block_name, primitive, height=28, x_cells=1, y_cells=1, p
         or primitive == 'generic', f"{block_name} definition: {primitive}"
 
     if primitive == 'generic':
-        uc, cell_pin = generate_generic(pdkdir, parameters, netlistdir=netlistdir)
```

Comment:
@soneryaldiz We don't seem to need `cell_pin` anymore. Should we remove it from the `generate_generic` return value?